deps/node-report/README.md

```diff
@@ -0,0 +1,128 @@
+# node-report
+
+Delivers a human-readable diagnostic summary, written to file.
+
+The report is intended for development, test and production
+use, to capture and preserve information for problem determination.
+It includes JavaScript and native stack traces, heap statistics,
+platform information and resource usage etc. With the report enabled,
+reports can be triggered on unhandled exceptions, fatal errors, signals
+and calls to a JavaScript API.
+
+Supports Node.js versions 4, 6 and 8 on AIX, Linux, MacOS, SmartOS and Windows.
+
+## Usage
+
+```bash
+npm install node-report
+node -r node-report app.js
```

Comment:
-r -> --report_events ?

Reply:
We'd drop this readme (but we do need to add some sort of documentation).